CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - Convert one of the two remaining usages of `sprintf` to `snprintf` (#2866)
+- Fix a use-after-free that could occur after initializing a UIImage with an empty name string (#3042)
```

Comment:
`m`: I think something like the following would be a more descriptive changelog; I don't think we need to explain the details.
```suggestion
- Fix use-after-free ASAN warning (#3042)
```